src/gossip/service.zig

```diff
@@ -1253,6 +1253,7 @@ pub const GossipService = struct {
                 _ = gossip_table.insert(req.value, now) catch {};
                 gossip_table.updateRecordTimestamp(req.value.id(), now);
             }
+            self.stats.table_n_updates_pull_req.add(pull_requests.items.len);
```

Comment:
`gossip_table.insert` will fail with an error if the value was not inserted which is not accounted for in this case -- would need to add logic to the catch block